docker-compose.yml

```diff
@@ -37,6 +42,20 @@ services:
       - network.host=0.0.0.0
       - bootstrap.memory_lock=true
       - "ES_JAVA_OPTS=-Xms1g -Xmx1g"
+      - ELASTICSEARCH_URL=https://elasticsearch:9200
+      - ELASTICSEARCH_IP=https://elasticsearch
+      - ELASTICSEARCH_PORT=9200
+      - SETUP_PASSWORDS=yes
+      - ADMIN_USER=admin_user
+      - ADMIN_PASS=admin_pass
+      - ELASTIC_PASSWORD=elastic_pass
+      - KIBANA_PASS=kibana_pass
+      - LOGSTASH_PASS=logstash_pass
+      - APM_SYSTEM_PASS=apmSystem_pass
+      - BEATS_SYSTEM_PASS=beatsSystem_pass
+      - REMOTE_USER_PASS=remoteUser_pass
+      - CA_PASS=ca_pass
+      - CERT_DNS=elasticsearch
```

Comment:
Change to `XPACK_SECURITY_CERTIFICATE_DNS`.

Reply:
Changed